helpers/validation.js

```diff
@@ -0,0 +1,21 @@
+async function isServerStaff(interaction, targetUser) {
+  const user = await interaction.guild.members.cache.get(targetUser.id);
+  return user?.roles?.cache.some((role) =>
+    [
+      process.env.ID_CODE_COUNSELOR,
+      process.env.ID_MODERATOR,
+      process.env.ID_ADMIN,
+      process.env.ID_SUPER_ADMIN,
+    ].includes(role.id)
+  )
+    ? true
+    : false;
```

Comment:
This works, but I don't think it needs the ternary.  It should be fine to just do something like
```js
return user?.roles?.cache.some((role) =>
    [
      process.env.ID_CODE_COUNSELOR,
      process.env.ID_MODERATOR,
      process.env.ID_ADMIN,
      process.env.ID_SUPER_ADMIN,
    ].includes(role.id))
    ```

Reply:
Good point on this. I removed the unnecessary ternary.